spec/system/admin/products_v3/create_spec.rb

```diff
@@ -120,4 +132,12 @@ def visit_products_page_as_admin
     login_as_admin
     visit spree.admin_products_path
   end
+
+  def new_variant_button
+    find("button.secondary.condensed.naked.icon-plus")
+  end
```

Comment:
In the previous spec, a different method, based on the button text is used. I think that would be better here, and it could be shared:
```
 page.find('button[aria-label="New variant"]', text: "New variant", visible: false)
```